test/controllers/EmailControllerSpec.scala

```diff
@@ -452,7 +456,7 @@ class EmailControllerSpec extends ItSpec {
                 Json.parse(
                   s"""
                    |{
-                   |  "origin" : "Bta",
+                   |  "origin" : "${origin.toString().takeRight(3)}",
```

Comment:
this might not be appropriate for other taxr egimes in the future - if you look at how the audit event is populating this value then you'd be able to do the same in the test here. This shoul be changed where it appears in other specs as well